### ISSUE TITLE
deploy bookinfo app in different namespace

### ISSUE THREAD

**Issue by debianmaster** [istio/issues#30]
i'm trying to deploy bookinfo app in a different namespace other than `default` where istio service mesh is deployed.   is there any special access needed?   i deployed non-auth version of istio.
and i see following errors on bookinfo app deployment

```console
Error syncing pod, skipping: [failed to "InitContainer" for "init" with RunInitContainerError: "init container \"init\" exited with 1" , failed to "StartContainer" for "init" with CrashLoopBackOff: "Back-off 5m0s restarting failed container=init pod=ratings-v1-1419296450-gglpr_istio-testing(7f4d39ac-506d-11e7-acf3-2cc2600c3cc4)" ]
```

**Comment by FuzzOli87** [comment]
https://istio.io/docs/tasks/installing-istio.html

Istio has to be deployed on the same namespace at the moment.

Also see:
https://github.com/istio/issues/issues/16

**Comment by rshriram** [comment]
This has been fixed as of 0.2.4.

**Comment by debianmaster** [comment]
I can confirm this works in latest
